src/pages/organize/[orgId]/campaigns/[campId]/calendar/tasks/[taskId]/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { GetServerSideProps } from 'next';
+import { useQuery } from 'react-query';
+import { Box, Container, Grid } from '@material-ui/core';
+
+import getTask from 'fetching/tasks/getTask';
+import { PageWithLayout } from 'types';
+import { scaffold } from 'utils/next';
+import SingleTaskLayout from 'components/layout/organize/SingleTaskLayout';
+import TaskDetailsSection from 'components/organize/tasks/TaskDetailsSection';
+
+const scaffoldOptions = {
+    authLevelRequired: 2,
+    localeScope: [
+        'layout.organize',
+        'misc.breadcrumbs',
+        'pages.organizeCampaigns',
+        'misc.formDialog',
+        'misc.tasks',
+        'misc.speedDial',
+    ],
+};
+
+export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const { orgId, campId, taskId } = ctx.params!;
+
+    await ctx.queryClient.prefetchQuery(
+        ['task', taskId],
+        getTask(
+            orgId as string,
+            taskId as string,
+            ctx.apiFetch,
+        ),
+    );
+    const taskQueryState = ctx.queryClient.getQueryState(['task', taskId]);
+
+    if (
+        taskQueryState?.status === 'success'
+    ) {
+        return {
+            props: {
+                campId,
+                orgId,
+                taskId,
+            },
+        };
+    }
+    else {
+        return {
+            notFound: true,
+        };
+    }
+}, scaffoldOptions);
+
+type TaskDetailPageProps = {
+    campId: string;
+    orgId: string;
+    taskId: string;
+}
+
+const TaskDetailPage: PageWithLayout<TaskDetailPageProps> = ({ taskId, orgId }) => {
+    const { data: task } = useQuery(['task', taskId], getTask(orgId, taskId));
+
+    if (!task) return null;
+
+    return (
+        <Box mt={ 4 }>
+            <Container>
+                <Grid container>
+                    <Grid item xs={ 12 }>
```

Comment:
I don't know if it's the grid, but the layout of the contents on this page is different from other pages. It seems to have a max width and is centered. On a wide screen it looks like this:

![image](https://user-images.githubusercontent.com/550212/126667144-01eff409-2830-4534-b7c2-65563f8a4480.png)


Is that intentional?

Reply:
Yea I just put it in a container for now, but would improve the layout more as more sections get added